README.md

```diff
@@ -35,6 +35,19 @@ Use labels to assist developers to know which image they should be using and whe
 - You will need to know what tag you are going to give to your new image, please read the [Tagging strategy](#tagging-strategy) section before continuing
 - It is desirable to add Labels to the docker image, please read through the [labelling docker images](#labelling-docker-images) section before continuing
 - If you are creating a completely new image that has not been used before, you will first need to create the Repository name in the AWS ECR Private registry - before building and pushing the image.
+- If you are using colima, ensure you have done the following:
+
+```bash
+brew install docker-buildx
```

Comment:
I think brew installs all plugins to `/usr/local/lib/docker/cli-plugins`
and then recommends making this change to the docker config file
```
        "cliPluginsExtraDirs": [
                "/usr/local/lib/docker/cli-plugins"
        ]
```
so that you don't need to link each plugin (as below), just get docker to look in that directory for all brew-installed ones

Reply:
Apparently (from the article i got the code from, and nope i did not make a not of the link) what i've used works on different versions of mac's (ones using M1 [which i think Jess has], that put things in different paths) and different versions of OS.